src/odemis/gui/util/img.py

```diff
@@ -2228,8 +2232,27 @@ def images_to_export_data(streams, view_hfw, view_pos,
     return (list of DataArray)
     raise LookupError: if no data visible in the selected FoV
     """
+    max_res = (MAX_RES_FACTOR * CROP_RES_LIMIT) ** 2
+    mpp = math.sqrt( (view_hfw[0]*view_hfw[1]) / max_res) # Area = [meters per pixel]^2 * number_of_pixels
```

Comment:
why min_mpp and not max_mpp or new_mpp for the new resolution? min_mpp suggests minimal_mpp

Reply:
Yes, `min_mpp`, because it's the smallest mpp we are willing to use (it would make the image the biggest we are willing to export). In many cases, the projection might not have such a small mpp.